Dockerfile

```diff
@@ -79,22 +125,36 @@ RUN pip3 install snakemake==5.10.0
 # from Google Storage URIs.
 RUN pip3 install google-cloud-storage==2.1.0
 
-# epiweeks (for ncov)
+# Install epiweeks (for ncov)
 RUN pip3 install epiweeks==2.1.2
 
-# Add Pangolin and PangoLEARN + deps (for ncov)
+# Install Pangolin and PangoLEARN + deps (for ncov)
 RUN pip3 install git+https://github.com/cov-lineages/pangolin.git@v3.1.17
 RUN pip3 install git+https://github.com/cov-lineages/pangoLEARN.git@2021-12-06
 RUN pip3 install git+https://github.com/cov-lineages/scorpio.git@v0.3.16
 RUN pip3 install git+https://github.com/cov-lineages/constellations.git@v0.1.1
 RUN pip3 install git+https://github.com/cov-lineages/pango-designation.git@19d9a537b9
 
-# Add Nextstrain components
+
+# 4. Add Nextstrain components
 
 # Allow caching to be avoided from here on out by calling
 # docker build --build-arg CACHE_DATE="$(date)"
 ARG CACHE_DATE
 
+# Nextclade/Nextalign are downloaded directly but using the latest version, so
+# they belong after CACHE_DATE.
+
+# Download Nextalign v2
+RUN curl -fsSL -o /out/bin/nextalign2 https://github.com/nextstrain/nextclade/releases/latest/download/nextalign-x86_64-unknown-linux-gnu
+# Set default Nextalign version to 2
+RUN ln -sv nextalign2 /out/bin/nextalign
+
+# Download Nextclade v2
+RUN curl -fsSL -o /out/bin/nextclade2 https://github.com/nextstrain/nextclade/releases/latest/download/nextclade-x86_64-unknown-linux-gnu
+# Set default Nextclade version to 2
+RUN ln -sv nextclade2 /out/bin/nextclade
```

Comment:
Why not bundle the `ln` calls here into the same `RUN`?

Reply:
Done: 8c3c27c9f5c6dc56863d040c29d60e8411dc4a17